### ISSUE TITLE
BUG: Album section runtime  issue . 

### ISSUE THREAD

**Issue by Abhiifour** [AOSSIE-Org/PictoPy#182]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### What happened?

when going to the album section , the screen goes blank white after sometime of loader spin . is it happening to all ?? 

https://github.com/user-attachments/assets/5a81f2e5-1861-43ad-b0b2-3cfe8c72df6a

### Record

- [X] I agree to follow this project's Code of Conduct

**Comment by agrawalshreyansh** [comment]
yes buddy, i have fixed that thing in Issue #163 , hopefully will be soon merged by @Pranav0-0Aggarwal
